internal/pkg/docker/dockercompose/decompose_test.go

```diff
@@ -122,31 +122,31 @@ func TestDecomposeService_General(t *testing.T) {
 			filename: "nginx-golang-postgres.yml",
 			svcName:  "db",
 
-			wantError: errors.New("\"services.db\" relies on fatally-unsupported Compose keys: secrets, volumes"),
+			wantError: errors.New("\"services.db\" relies on fatally-unsupported Compose keys: secrets"),
 		},
 		"nginx-golang-postgres proxy": {
 			filename: "nginx-golang-postgres.yml",
 			svcName:  "proxy",
 
-			wantError: errors.New("\"services.proxy\" relies on fatally-unsupported Compose keys: volumes"),
+			wantError: errors.New(`convert Compose service to Copilot manifest: volume type "bind" is not supported yet`),
 		},
 		"react-express-mongo frontend": {
 			filename: "react-express-mongo.yml",
 			svcName:  "frontend",
 
-			wantError: errors.New("\"services.frontend\" relies on fatally-unsupported Compose keys: networks, volumes"),
+			wantError: errors.New("\"services.frontend\" relies on fatally-unsupported Compose keys: networks"),
```

Comment:
you can add the backticks to the other test cases as well :)

Reply:
Done! No more `\"` in this package 😄